README.md

```diff
@@ -127,16 +127,24 @@ With a Mongo database set up, ensure that the following environment variables ex
 - `MONGO_CONNECTION_STRING`: The connection string for your MongoDB
 - `DATABASE_NAME`: The name of the database to be used (It will be created if it doesn't exist)
 
+## Oauth
+This application uses Github Oauth to authorise users. Ensure that you have registered the app with Github Oauth
+(`settings > developer settings > Oauth apps`) and added the appropriate secrets to your `.env` file.
```

Comment:
A minor one but you've documented the environment variables for Mongo, so I'd also document the Oauth ones here too rather than assuming they are known.